### DISCUSSION TITLE
Welcome to Tanka Discussions!

### DISCUSSION THREAD

**Discussion by tombrk** [grafana/tanka#442] [General]
<p align="center">
  <img
    width="400"
    src="https://raw.githubusercontent.com/grafana/tanka/master/docs/img/logo.svg"
    alt="Grafana Tanka Logo"
  />
</p>
<h1 align="center">Tanka Discussions</h1>

Welcome! :wave:

The Tanka project is trying out GitHub Discussions as the **primary support channel**:

* **Ask your questions** to the team and the community.  
Please make sure to ask a [good question](https://stackoverflow.com/help/how-to-ask) to receive a good answer :stuck_out_tongue: 
* **Get early feedback for ideas**: If your idea is not ready enough for a proposal as an issue yet, feel free to chat about it here.

**Comment by lpriori-cell** [comment]
Will an analog of helm hooks be added, there is a great need for this, at least pre-install and pre-delete hooks, even grafana charts need hooks (example: k8s-monitoring)

**Reply by zerok** [reply]
Hey 👋  Something like that is not on our roadmap, but a more detailed issue about this is definitely welcome. Makes it easier to discuss the pros and cons 🙂
